alpakkaCassandra/src/main/resources/reference.conf

```diff
@@ -0,0 +1,21 @@
+alpakka.cassandra.session {
```

Comment:
Not used to having camelCase in directory names. It would be better to have `alpakka-cassandra`. But not really important as it will move to Alpakka repo anyways.